contracts/standard/arbitration/composed-arbitrable/example/ArbitrableTokenList.sol

```diff
@@ -0,0 +1,594 @@
+        require(!agreement.disputed, "The item is still disputed.");
+        require(agreement.creator != address(0), "The specified agreement does not exist.");
+        require(!agreement.executed, "The specified agreement has already been executed.");
+        require(!agreement.disputed, "The specified agreement is disputed.");
+        agreement.executed = true;
+
+        if (item.status == ItemStatus.Resubmitted || item.status == ItemStatus.Submitted)
+            item.status = ItemStatus.Registered;
+        else if (item.status == ItemStatus.ClearingRequested || item.status == ItemStatus.PreventiveClearingRequested)
+            item.status = ItemStatus.Cleared;
+        else
+            revert("Item in wrong status for executing request.");
+
+        item.challengeReward = 0; // Clear challengeReward once a dispute is resolved.
+        item.lastAction = now;
+        agreement.parties[0].send(item.balance); // Deliberate use of send in order to not block the contract in case of reverting fallback.
+        item.balance = 0;
+
+        emit ItemStatusChange(agreement.parties[0], agreement.parties[1], _tokenID, item.status, agreement.disputed);
+    }
+
+    /** @dev Changes the `challengeReward` storage variable.
+     *  @param _challengeReward The new `stake` storage variable.
+     */
+    function changeChallengeReward(uint _challengeReward) public {
+        require(msg.sender == challengeRewardGovernor, "The caller is not the fee governor.");
+        challengeReward = _challengeReward;
+    }
+
+    /** @dev Changes the `challengeRewardGovernor` storage variable.
+     *  @param _challengeRewardGovernor The new `challengeRewardGovernor` storage variable.
+     */
+    function changeChallengeRewardGovernor(address _challengeRewardGovernor) public {
+        require(msg.sender == challengeRewardGovernor, "The caller is not the fee governor.");
+        challengeRewardGovernor = _challengeRewardGovernor;
+    }
+
+    /* Public Views */
+
+    /**
+     *  @dev Returns the latest agreement for an item
+     *  @param _tokenID The tokenID of the item to check.
+     *  @return The latest agreementID
+     */
+    function latestAgreementId(bytes32 _tokenID) public view returns (bytes32) {
+        return keccak256(abi.encodePacked(_tokenID, itemIDToAgreementCount[_tokenID]));
+    }
+
+    /**
+     *  @dev Return true if the item is allowed.
+     *  We consider the item to be in the list if its status is contested and it has not won a dispute previously.
+     *  @param _tokenID The tokenID of the item to check.
+     *  @return allowed True if the item is allowed, false otherwise.
+     */
+    function isPermitted(bytes32 _tokenID) public view returns (bool allowed) {
+        Item storage item = items[_tokenID];
+        Agreement storage latestAgreement = agreements[latestAgreementId(_tokenID)];
+        bool _excluded = item.status <= ItemStatus.Resubmitted ||
```

Comment:
This will show pending items and disputed preemptive clearing requests items as permitted.

We should just return true if the status is "Registered" or "ClearingRequested", and false if it is anything else.